mkdocs/docs/start.md

```diff
@@ -81,10 +81,18 @@ which can be directly feed into the `fit` function of `ImageClassifier`.
 
     from autokeras import ImageClassifier
     clf = ImageClassifier(verbose=True, augment=False)
-    clf.load_searcher().load_best_model().produce_keras_model().save('my_model.h5')
+    clf.export_autokeras_model(model_file_name)
```

Comment:
This is not correct. It is export keras model.
You should keep the original and write a new section "how to export Auto-Keras model"

Reply:
This has been fixed. Add another section as:
* How to export Portable model
* How to load exported Portable model?